contrib/install.sh

```diff
@@ -28,9 +29,18 @@ parse_args() {
   # over-ridden by flag below
 
   BINDIR=${BINDIR:-./bin}
-  while getopts "b:dh?x" arg; do
+  CLIENT=${CLIENT:-install-script}
+  while getopts "b:c:dh?x" arg; do
     case "$arg" in
       b) BINDIR="$OPTARG" ;;
+      c)
+        if printf '%s' "$OPTARG" | grep -Eq '^[A-Za-z0-9-]+$'; then
```

Comment:
Is the client length important for us?
We are going to process this information anyway, right?

Reply:
its doesn't really matter. The client is handled in the download api. As long as it's not some sort of SQL injection, length doesn't matter.